docs/_includes/common/listbox.html

```diff
@@ -114,4 +114,57 @@ <h3 id="listbox-selected">Selected State</h3>
     </div>
 </div>
     {% endhighlight %}
+    <h3 id="listbox-selected">Long Option</h3>
```

Comment:
This should be a test case / edge case under storybook only. No need for it to be on the main docs.